.gitignore

```diff
@@ -26,6 +26,7 @@ catalog-services/catalog-services.iml
 
 # local docker volume
 docker/local-metadata/docker-volume
+docker-volume
```

Comment:
@akash-jain-10 @Vj-L do we want to add this in gitignore?

Reply:
Yes! Lets keep this for now.